sdk/core/ts-http-runtime/review/ts-http-runtime.api.md

```diff
@@ -280,15 +116,6 @@ export function getClient(endpoint: string, options?: ClientOptions): Client;
 // @public
 export function getClient(endpoint: string, credentials?: TokenCredential | KeyCredential, options?: ClientOptions): Client;
```

Comment:
I wonder if we could benefit from making this overload to have credentials as required? If you don't want to pass credentials the first can be used

Reply:
Yeah we totally could. I think we could do that for Azure too actually, don't think it would be a breaking change. I'll open an issue for it

Reply:
Opened https://github.com/Azure/azure-sdk-for-js/issues/32023 -- it would be a breaking change for Azure if we made `credentials` required, but no reason we couldn't do that for unbranded (other than consistency)